src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,126 @@
+package duke;
+
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
+    private static final TasksList tasks = new TasksList();
+
+    public static void main(String[] args) {
+        String rawInput;
+        Scanner scanner = new Scanner(System.in);
+        DukeCommand dukeCommand;
+
+        try {  // Initialize ./data/duke.txt to store tasks
+            tasks.initDataStore();
+        } catch (IOException e) {
+            System.out.println("\tDirectory does not exist, error in loading data store.\n" +
```

Comment:
Avoid the use of magic strings. In this case, the error message can be extracted into a constant variable with a suitable name.